condor_watch_q.py

```diff
@@ -449,6 +448,7 @@ def process_events(self):
                     ),
                 )
 
+
```

Comment:
Please remove spurious whitespace-only changes (there are a few others in this PR to fix as well)